LoopKitUI/Views/MultiButtonTableFooterView.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  MultiButtonTableFooterView.swift
+//  LoopKitUI
+//
+//  Created by Nathaniel Hamming on 2020-01-27.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import UIKit
+
+public class MultiButtonTableFooterView: UIView {
```

Comment:
This seems to be serving the same role as SetupTableViewController; please extend SetupTableViewController instead.

Reply:
I find the name `SetupTableViewController` is too specific. So I'll refactor with this in mind.

Reply:
After thinking about this more, I've updated the `SetupTableFooterView`. We can update with design language at a later date once this is better known.